### PR TITLE
B lorient547

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient547.xml [hunk 1]
```diff
@@ -107,13 +89,204 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+
+<msItem xml:id="p1_i2.1.2">
+<locus from="28r"/>
+<title xml:lang="en" type="complete" ref="LIT1960Mashaf#SerataQeddase">The ordinary Canon of the Mass</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="p1_i2.1.3">
+<locus from="68r"/>
+<title xml:lang="en" type="complete" ref="LIT3189Anapho">Anaphoras</title>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient547.xml:155]: LIT1960Mashaf#Anaphoras, if it is a collection, it is much better to use the second-level ids

File: LondonBritishLibrary/orient/BLorient547.xml [hunk 2]
```diff
@@ -107,13 +89,204 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="p1_i2.1.3.1.1">
+<locus from="68r"/>
+<title xml:lang="en" type="complete" ref="LIT1104Anapho">Anaphora of the apostles</title>
+<textLang mainLang="gez"/>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient547.xml:162]: LIT1960Mashaf#AnaphoraApostles

File: LondonBritishLibrary/orient/BLorient547.xml [hunk 3]
```diff
@@ -107,13 +89,204 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+
+  <item xml:id="a3">
+  <locus target="#4r"></locus>
+    <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+            <placeName ref="INS0101MadhaneAlam"/>.</desc>
+        <q xml:lang="gez">ቅዳሴ፡ ዘ<placeName ref="INS0101MadhaneAlam">ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName></q>
+  </item>
+
+  <item xml:id="e1">
+  <desc type="MixedNote">The name <persName ref="PRS13414Walatta_">Walatta Giyorgis</persName> occurs
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient547.xml:279]: I do not think that the type is needed here.

File: LondonBritishLibrary/orient/BLorient547.xml (hunk 1 above)
Comment (DariaElagina) [reply LondonBritishLibrary/orient/BLorient547.xml:155]: This is applicable for all the following msItems

File: LondonBritishLibrary/orient/BLorient547.xml [hunk 4]
```diff
@@ -107,13 +89,204 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+<msIdentifier></msIdentifier>
+
+<msContents>
+
+<msItem xml:id="p1_i2.1">
+<locus from="4r"/>
+<title xml:lang="en" type="complete" ref="LIT1960Mashaf">The missal</title>
+<notatedMusic><desc>Musical notes have been added.</desc></notatedMusic>
+<textLang mainLang="gez"/>
+</msItem>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient547.xml:139]: Please, pay attention to the nesting, the following msItems should be contained within this one.

File: LondonBritishLibrary/orient/BLorient547.xml [hunk 5]
```diff
@@ -107,13 +89,204 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+<locus from="28r"/>
+<title xml:lang="en" type="complete" ref="LIT1960Mashaf#SerataQeddase">The ordinary Canon of the Mass</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="p1_i2.1.3">
+<locus from="68r"/>
+<title xml:lang="en" type="complete" ref="LIT3189Anapho">Anaphoras</title>
+<textLang mainLang="gez"/>
+</msItem>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient547.xml:157]: Here there is also a problem with the hierarchy (nesting)

**Review by DariaElagina — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient547.xml [hunk 6]
```diff
@@ -1,12 +1,11 @@
...
+?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
    schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:lang="en" xml:id="BLorient547" type="mss">
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title>Qǝddāse or Missal
-                </title>
+                <title>Qǝddāse</title>
                 <editor key="SG"/>
                 <editor key="AB" role="generalEditor"/>
```
[the remark this answers is not in the document]
Comment (MarcinKrawczuk) [reply LondonBritishLibrary/orient/BLorient547.xml:10]: Done.

File: LondonBritishLibrary/orient/BLorient547.xml [hunk 7]
```diff
@@ -107,13 +89,204 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+
+<msPart xml:id="p1">
+<msIdentifier></msIdentifier>
+
+<msContents>
+
+<msItem xml:id="p1_i1">
+<locus from="2r"/>
+<title xml:lang="en" type="incomplete">Confession of faith</title>
+<note>Incomplete at the beginning.</note>
```
[the remark this answers is not in the document]
Comment (MarcinKrawczuk) [reply LondonBritishLibrary/orient/BLorient547.xml:106]: If you say so :) But maybe with a textpart?

File: LondonBritishLibrary/orient/BLorient547.xml (hunk 4 above)
Comment (MarcinKrawczuk) [reply LondonBritishLibrary/orient/BLorient547.xml:139]: Sorry :( Corrected.

File: LondonBritishLibrary/orient/BLorient547.xml (hunk 5 above)
Comment (MarcinKrawczuk) [reply LondonBritishLibrary/orient/BLorient547.xml:157]: OK, I hope it makes more sense now.

File: LondonBritishLibrary/orient/BLorient547.xml (hunk 3 above)
Comment (MarcinKrawczuk) [reply LondonBritishLibrary/orient/BLorient547.xml:279]: sure

**Review by DariaElagina — APPROVED** [review]

**Comment by DariaElagina** [comment]
Thank you! I will delete it from the list!

**Pull request merged**